recipes-kernel/linux/linux-ae350-ax45mp-dev.bb

```diff
@@ -0,0 +1,18 @@
+require linux-mainline-common.inc
+SUMMARY = "AE350 AX45MP dev kernel recipe"
+
+LIC_FILES_CHKSUM = "file://COPYING;md5=6bc538ed5bd9a7fc9398086aedcd7e46"
+KERNEL_VERSION_SANITY_SKIP = "1"
+
+SRCREV = "${AUTOREV}"
```

Comment:
I see this. is a dev kernel and perhaps AUTOREV is fine but lets make that a suggestion for someone needing it and let the defaults be some real SHA

Reply:
The default AUTOREV is defined.

Thanks.